lnd.go

```diff
@@ -182,10 +181,9 @@ func lndMain() error {
 	}
 	sCreds := credentials.NewTLS(tlsConf)
 	serverOpts := []grpc.ServerOption{grpc.Creds(sCreds)}
-	grpcEndpoint := fmt.Sprintf("localhost:%d", loadedConfig.RPCPort)
-	restEndpoint := fmt.Sprintf(":%d", loadedConfig.RESTPort)
-	cCreds, err := credentials.NewClientTLSFromFile(cfg.TLSCertPath,
-		"")
+	grpcEndpoint := cfg.RPCListeners[0]
```

Comment:
It should be documented in the help for the config option that for the initial wallet creation/unlocking, only the _first_ listener will be used. 

On the other hand, this seems like an artificial restriction, why not just use them all?

Reply:
I updated `waitForWalletPassword()` to listen on all the same RPC/REST interfaces.